src/metabase/models/dashboard_card_series.clj

```diff
@@ -12,3 +13,23 @@
 
 (doto :model/DashboardCardSeries
   (derive :metabase/model))
+
+;; Serialization
+
+(defmethod serdes/entity-id "DashboardCardSeries" [_ instance] (:card_id instance))
+
+(defmethod serdes/generate-path "DashboardCardSeries" [_ _] nil)
+
+(defmethod serdes/load-find-local "DashboardCardSeries" [path]
+  (let [{:keys [id]} (last path)]
+    (t2/select-one :model/DashboardCardSeries :card_id {:from   [:report_card]
+                                                        :select [:id]
+                                                        :where  [:= :entity_id id]})))
```

Comment:
This will only select one `DashboardCardSeries` for a card. But there could be many dashboard card series for a given card.